include/ignition/rendering/LidarVisual.hh

```diff
@@ -179,6 +179,14 @@ namespace ignition
       /// \brief Get if non-hitting rays will be displayed
       /// \return Boolean value if non-hitting rays will be displayed
       public: virtual bool DisplayNonHitting() const = 0;
+
+      /// \brief Set visibility of the visual
+      /// \param[in] _visible Visibility of the visual
+      public: virtual void SetVisible(bool _visible) = 0;
```

Comment:
LidarVisual should inherit the `SetVisible` function from Visual class already so you should not have to add a new one here